contrib/connector/iscsi/helper.go

```diff
@@ -248,14 +248,18 @@ func connect(connMap map[string]interface{}) (string, error) {
 
 	targetlun := strconv.Itoa(conn.TgtLun)
 
-	cmd := "pgrep -f /sbin/iscsid"
-	_, err = connector.ExecCmd("/bin/bash", "-c", cmd)
+    cmd := "ls -ali / | sed '2!d' |awk {'print $1'}"
```

Comment:
Any particular reason to use 'al'? All listing can be large at times and may not be efficient.